src/lib/validation/validate.ts

```diff
@@ -74,6 +74,12 @@ export const getFormErrors = async (
   }
 };
 
+function isScalarValue(obj: IValues | Value | Values): obj is Value {
+  if (Array.isArray(obj)) return false;
+  if (obj == null) return true;
```

Comment:
Can this be a strict comparison?

Reply:
Yep, I was thrown off by the convention that `foo == null` catches both `null` and `undefined`, but in this instance that's not relevant since `typeof undefined` is `undefined` so we only need to check for null explicitly. Addressed in #29